resources/resources.md

```diff
@@ -0,0 +1,16 @@
+---
+nav_order: 121
+has_children: true
+title: Resources
+description: Machine translation resources
+---
+
+### Calls for papers
+
+| Publication | Organisers | Deadline | Type |
```

Comment:
If we make the publications italics (which we should), and leave the conferences without italics (which is fine, I guess?), then we can remove the "type" column and save lots of space. :-)

Reply:
@bittlingmayer Are we really about to make a contract with ourselves to update this page periodically? There are lots of workshops and conferences and keeping track of them all may be problematic.